packages/core/content-manager/server/src/history/models/history-version.ts

```diff
@@ -0,0 +1,48 @@
+import type { Model } from '@strapi/database';
+import { HISTORY_VERSION_UID } from '../constants';
+
+const historyVersion: Model = {
+  uid: HISTORY_VERSION_UID,
+  tableName: 'strapi_history_versions',
+  singularName: 'history-version',
+  attributes: {
+    id: {
+      type: 'increments',
+    },
+    contentType: {
+      type: 'string',
+      column: { notNullable: true },
+    },
+    // documentId is a reserved attribute name
+    relatedDocumentId: {
+      type: 'string',
+      column: { notNullable: true },
+    },
+    locale: {
+      type: 'string',
+    },
+    status: {
+      type: 'enumeration',
+      enum: ['draft', 'published', 'modified'],
+    },
+    data: {
+      type: 'json',
+    },
+    schema: {
+      type: 'json',
+    },
+    createdAt: {
+      type: 'datetime',
+    },
+    // FIXME: joinTable should be optional
+    // @ts-expect-error database model is not yet updated to support useJoinTable
+    createdBy: {
+      type: 'relation',
+      relation: 'oneToOne',
+      target: 'admin::user',
+      useJoinTable: false,
+    },
```

Comment:
is it necessary to use a join table for the created by field?

Reply:
Do you think it would be? Here it's false, and for normal content types it's false as well

Reply:
didn't select the right section 😓 I think the attribute is correctly set up and will not use a join table, but there is a message about jointable being optional and didn't get why 🤔

Reply:
Ah I see. The comment is because TS wanted us to provide a joinTable property (not sure about the name exactly). But it shouldn't be required when useJoinTable is false. The type isn't smart enough for that yet, that's why we expect a TS error here